pom.xml

```diff
@@ -48,7 +48,7 @@
 		<dependency>
 			<groupId>com.amazonaws</groupId>
 			<artifactId>aws-java-sdk-cloudformation</artifactId>
-			<version>1.9.27</version>
+			<version>1.10.62</version>
```

Comment:
This change is not topical to the commit. I would much rather have this update be tied to a ticket in JIRA specifically stating to update the SDK version to the latest version.